app/controllers/api/v3/scenarios_controller.rb

```diff
@@ -118,10 +118,12 @@ def create
         # Weird ActiveResource bug: the user values attribute is nested inside
         # another user_values hash. Used when generating a scenario with the
         # average values of other scenarios.
-        inputs = params[:scenario][:user_values]["user_values"] rescue nil
-        if inputs
-          params[:scenario][:user_values] = inputs
+        inputs = begin
+          params[:scenario][:user_values]['user_values']
+        rescue StandardError
+          nil
         end
+        params[:scenario][:user_values] = inputs if inputs
```

Comment:
Why did you change this?

Reply:
It's all rubocop... gonna need to turn it off I think 😅. I will try revert the rubocop changes in another commit.